Solutions/Dev 0270 Detection and Hunting/Analytic Rules/Dev-0270PowershellSep2022.yaml

```diff
@@ -26,7 +26,7 @@ query: |
   (union isfuzzy=true
   (SecurityEvent
   | where EventID==4688
-  | extend FileName=tostring(split(NewProcessName, @'')[(-1)]),  ProcessCommandLine = CommandLine, InitiatingProcessFileName=ParentProcessName
```

Comment:
I would suggest project 'InitiatingProcessFileName' instead of removing from 'SecurityEvent' Section. 'InitiatingProcessFileName' column is being used in the other table 'DeviceProcessEvents '. Just to keep similar columns.

Reply:
the other 'Split' change looks good.